beetmoverscript/script.py

```diff
@@ -86,6 +95,46 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if update_balrog_manifest:
+        context.balrog_manifest.append(
+            enrich_balrog_manifest(context, abs_file_path, locale, destinations)
+        )
+
+
+def enrich_balrog_manifest(context, path, locale, destinations):
+    props = context.properties
+
+    if props["branch"] == 'date':
```

Comment:
yeah, I didn't like hardcoding anything branch specific. thanks for pointing this out.